src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class ColorSupplier {
+    private String color;
+
+    public String getRandomColor() {
+
+        int randomColorNumber = (int) Math.random() * 3;
+
+        switch (randomColorNumber) {
+            case 0: {
+                color = "Red";
+            }
+            break;
+            case 1: {
+                color = "Green";
+            }
+            break;
```

Comment:
```suggestion
            case 1: {
                return "Green";
            }
```
можна одразу повертати значення з свіча